### PR TITLE
[-] BO : Fix #PSCFV-8002 controller employee access

### PR DESCRIPTION
**Pull request by sfroment**
http://forge.prestashop.com/browse/PSCFV-8022
http://forge.prestashop.com/browse/PSCFV-8002
http://forge.prestashop.com/browse/PSCSX-2077
http://forge.prestashop.com/browse/PSCSX-3994

### CODE REVIEW

**Comment by florent33700** [comment]
Hello,
Your changes do not fully correct the problem.
First, a shop x can see its customer addresses but can not edit them. Indeed, when you click on an address, you get the message "Access denied"; problem that did not exist before the correction.
On the other hand, the changes you made only correct display of the view. For example, it is still possible for a shop x to remove addresses of an other shop by simply entering an URL like :
index.php?controller=AdminAddresses&id_address=1&deleteaddress&token=
In conclusion, the user visible part is protected but the code behind can still be executed.

**Comment by sfroment42** [comment]
Hello,

Thanks for feedback.
I close the PR for the moment and I will reopen an other when it will be fixed

Best regards.

**Comment by florent33700** [comment]
When do you think it will be fixed ? Are you working on it ?
